### PR TITLE
[IMP] project: simplify task stage creation check in kanban view

### PR DESCRIPTION
**Pull request by mjvi-odoo**
- Remove the use of `active_model`​ in the context of `act_project_project_2_project_task_all` action.

task-4134660

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/project/static/src/views/project_task_kanban/project_task_kanban_renderer.js [hunk 1]
```diff
@@ -19,17 +19,9 @@ export class ProjectTaskKanbanRenderer extends KanbanRenderer {
...
             && this.isProjectManager || this.props.list.groupByField.name === 'personal_stage_type_id');
     }
-
-    isProjectTasksContext() {
-        return (
-            ["project.project", "project.task.type.delete.wizard"].includes(
-                this.props.list.context.active_model
-            ) && !!this.props.list.context.default_project_id
-        );
-    }
```
Comment (ppr-odoo) [inline addons/project/static/src/views/project_task_kanban/project_task_kanban_renderer.js:32]: If this method is used in customization, what will happen?

**Review by auon-odoo — APPROVED** [review]
lgtm :fire:

File: addons/project/static/src/views/project_task_kanban/project_task_kanban_renderer.js [hunk 2]
```diff
@@ -23,15 +23,11 @@ export class ProjectTaskKanbanRenderer extends KanbanRenderer {
...
-
-    isProjectTasksContext() {
         return (
-            ["project.project", "project.task.type.delete.wizard"].includes(
-                this.props.list.context.active_model
-            ) && !!this.props.list.context.default_project_id
+            super.canCreateGroup() &&
+            ((!!this.props.list.context.default_project_id == this.props.list.isGroupedByStage &&
+                this.isProjectManager) ||
+                this.props.list.groupByField.name === "personal_stage_type_id")
```
Comment (auon-odoo) [inline addons/project/static/src/views/project_task_kanban/project_task_kanban_renderer.js:30]:
```suggestion
            super.canCreateGroup() && ((
                !!this.props.list.context.default_project_id == this.props.list.isGroupedByStage && this.isProjectManager
            ) || this.props.list.groupByField.name === "personal_stage_type_id")
```
Looks more readable to me, but it doesn't really matter :shrug:

**Review by auon-odoo — APPROVED** [review]
Looks good to me! :fire:

**Review by xavierbol — APPROVED** [review]
LGTM, thanks for your work! 🙂

**Comment by xavierbol** [comment]
robodoo r+

**Pull request closed without merging**